src/pages/icp-tokens.tsx

```diff
@@ -0,0 +1,757 @@
+              <div className="md:w-7/10">
+                <motion.h1
+                  className="tw-heading-3 md:tw-heading-2 mb-8 md:mb-6 relative"
+                  variants={transitions.item}
+                >
+                  Understanding the ICP token{" "}
+                </motion.h1>
+                <motion.p
+                  className="tw-lead-sm md:tw-lead mb-0 relative"
+                  variants={transitions.item}
+                >
+                  Engage with the Internet Computer and its ecosystem through
+                  the use of its native utility token.
+                </motion.p>
+              </div>
+            </AnimateSpawn>
+          </section>
+
+          <AnimateSpawn
+            el={motion.section}
+            variants={transitions.container}
+            className="container-10 relative md:pt-52"
+          >
+            <div className="-mt-40 md:mt-0 md:absolute md:right-0 md:-top-60 text-center">
+              <motion.img
+                variants={transitions.fadeIn}
+                src="/img/icp-tokens/hero.webp"
+                alt=""
+                className="w-full max-w-sm md:max-w-none aspect-[563/492]"
+              />
+            </div>
+            <div className="mb-12 md:w-7/10">
+              <motion.h2
+                className="tw-heading-4 md:tw-heading-3 mb-6 text-gradient"
+                variants={transitions.item}
+              >
+                Powers computation, staking, voting, governance & ownership.{" "}
+              </motion.h2>
+              <motion.p
+                className="tw-paragraph md:tw-lead-sm text-black/60 mb-3"
+                variants={transitions.item}
+              >
+                The ICP token plays a number of roles on the Internet Computer.
+                Firstly, as a governance token, it can be staked to exercise
+                those governance rights. As a utility token, it can be burned to
+                obtain ‘cycles‘ which serves as gas for computation and storage
+                in canister smart contracts. It can also be minted to reward
+                ‘node machine‘ providers for providing that compute and storage.
+              </motion.p>
+              <motion.p
+                className="tw-paragraph md:tw-lead-sm text-black/60 mb-3"
+                variants={transitions.item}
+              >
+                In addition to these core uses, ICP tokens can be used to
+                participate in decentralisation swaps to become a co-owner of an
+                SNS DAO, and when using many smart contract services built on
+                the Internet Computer such as registries, marketplaces and
+                exchanges.
+              </motion.p>
+              <motion.p
+                className="tw-paragraph md:tw-lead-sm text-black/60 mb-0"
+                variants={transitions.item}
+              >
+                The ICP token implements the ICRC-1 standard.
```

Comment:
ICP is *compatible* with the ICRC-1 standard, but it uses a different account format and stores block data differently: https://internetcomputer.org/docs/current/developer-docs/defi/tokens/ledger/overview#icp-ledger

Reply:
Thanks @jessiemongeon1, feel free to update directly here on this PR